charts/keycloak/templates/route.yaml

```diff
@@ -30,5 +30,17 @@ spec:
   tls:
     insecureEdgeTerminationPolicy: {{ $route.tls.insecureEdgeTerminationPolicy }}
     termination: {{ $route.tls.termination }}
+    {{- if $route.tls.certificate }}
```

Comment:
Sorry i forgot about this in the last review... Please also add the new values to the README

Reply:
Can you mark this change as done? :) I would love to have this in the next Chart Release.